### PR TITLE
Add Environment Update Script

### PR DESCRIPTION
**Pull request by mfixstsci**
This is the final portion of the release process. This script allows user to download the environment used in a release, install it and set it as the environment `jwql-current`.

General Use Case:
`source install-env.sh <tagged_version> <python_version>`

Specific Use Case:
`source install-env.sh v1.2.9-a.1 3.11`

### CODE REVIEW

**Comment by york-stsci** [comment]
The script (and its underlying concept) look good to me. Will approve after seeing it in action on dev.

**Comment by mfixstsci** [comment]
@york-stsci We did the demo a few weeks ago and was hoping I could get approval. @BradleySappington and I are going to perform a release next Wednesday at 3PM EST if you would like to join us.

**Review by york-stsci — APPROVED** [review]
Looks good to me.

**Pull request merged**
